doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2409,3 +2409,55 @@ setting indicates embargoes are not supported. A value of -1 allows embargoes of
 can enter for an embargo end date. This limit will be enforced in the popup dialog in which users enter the embargo date. For example, to set a two year maximum:
 
 ``curl -X PUT -d 24 http://localhost:8080/api/admin/settings/:MaxEmbargoDurationInMonths``
+
+:DataverseMetadataValidatorScript
++++++++++++++++++++++++++++++++++
+
+An optional external script that validates Dataverse collection metadata as it's being updated or published. The script provided should be an executable that takes a single command line argument, the name of the file containing the metadata exported in the native json format. The script should exit with a non-zero error code if the validation fails. If that happens, a failure message (customizable in the next two settings below, `:DataverseMetadataValidationFailureMsg` and `DataverseMetadataUpdateValidationFailureMsg`) will be shown to the user.
+
+For example:
+
+``curl -X PUT -d /usr/local/bin/dv_validator.sh http://localhost:8080/api/admin/settings/:DataverseMetadataValidatorScript``
+
+
+:DataverseMetadataValidationFailureMsg
```

Comment:
:DataverseMetadataValidationPublishFailureMsg might be more clear as a setting name.

Otherwise, the only difference between this setting and the one below is the presence or absence of the word "update".

Reply:
it's pure legacy. we thought it would only need to be done on publish, initially, just like for datasets. Then remembered that there is no "draft" published dataverses - so somebody could publish an innocuous dataverse, then add spam text to the description...